Microsoft.Identity.Web.UI/Microsoft.Identity.Web.UI.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
```

Comment:
>PropertyGroup [](start = 3, length = 13)

As discussed, We can remove addition of another library (Microsoft.Identity.Web.UI), by moving its contents to the existing library , Microsoft.Identity.Web . Microsoft.Identity.Web will be converted to a Razor class library.